docs/source/prereqs.rst

```diff
@@ -75,126 +74,11 @@ following command from a terminal prompt:
-example for ``bash`` under Linux:
-
-.. code:: bash
-
-  export PATH=$PATH:$GOPATH/bin
-
-While this directory may not exist in a new Go workspace installation, it is
-populated later by the Fabric build system with a small number of Go executables
-used by other parts of the build system. So even if you currently have no such
-directory yet, extend your shell search path as above.
-
-Node.js Runtime and NPM
------------------------
-
-If you will be developing applications for Hyperledger Fabric leveraging the
-Hyperledger Fabric SDK for Node.js, version 8 is supported from 8.9.4 and higher.
-Node.js version 10 is supported from 10.15.3 and higher.
-
-  - `Node.js <https://nodejs.org/en/download/>`__ download
-
-.. note:: Installing Node.js will also install NPM, however it is recommended
-          that you confirm the version of NPM installed. You can upgrade
-          the ``npm`` tool with the following command:
-
-.. code:: bash
-
-  npm install npm@5.6.0 -g
-
-Python
-^^^^^^
-
-.. note:: The following applies to Ubuntu 16.04 users only.
-
-By default Ubuntu 16.04 comes with Python 3.5.1 installed as the ``python3`` binary.
-The Fabric Node.js SDK requires an iteration of Python 2.7 in order for ``npm install``
-operations to complete successfully.  Retrieve the 2.7 version with the following command:
-
-.. code:: bash
-
-  sudo apt-get install python
-
-Check your version(s):
-
-.. code:: bash
-
-  python --version
-
 .. _windows-extras:
 
 Windows extras
 --------------
 
-If you are developing on Windows 7, you will want to work within the
-Docker Quickstart Terminal. However, by default it uses an old `Git
-Bash <https://git-scm.com/downloads>`__ and experience has shown this
-to be a poor development environment with limited functionality. It is
-suitable to run Docker based scenarios, such as
-:doc:`getting_started`, but you will have difficulties with operations
-involving the ``make`` and ``docker`` commands.
-
-Instead, it is recommended to use the MSYS2 environment and run make
-and docker from the MSYS2 command shell. To do so, `install
-MSYS2 <https://github.com/msys2/msys2/wiki/MSYS2-installation>`__
-(along with the base developer toolchain and gcc packages using
-pacman) and launch Docker Toolbox from the MSYS2 shell with the
-following command:
-
-::
-
-   /c/Program\ Files/Docker\ Toolbox/start.sh
-
-Alternatively, you can change the Docker Quickstart Terminal command
-to use MSYS2 bash by changing the target of the Windows shortcut from:
-
-::
-
-   "C:\Program Files\Git\bin\bash.exe" --login -i "C:\Program Files\Docker Toolbox\start.sh"
-
-to:
-
-::
-
-   "C:\msys64\usr\bin\bash.exe" --login -i "C:\Program Files\Docker Toolbox\start.sh"
-
-With the above change, you can now simply launch the Docker Quickstart
-Terminal and get a suitable environment.
-
 On Windows 10 you should use the native Docker distribution and you
 may use the Windows PowerShell. However, for the ``binaries``
```

Comment:
Same question and comments as above. Not sure what the ``binaries`` command is and changing the global config is not always a good idea.